activerecord/lib/active_record/readonly_attributes.rb

```diff
@@ -15,7 +15,9 @@ module ClassMethods
       # Attributes listed as readonly will be used to create a new record but update operations will
       # ignore these fields.
       #
-      # You can assign a new value to a readonly attribute, but it will be ignored when the record is updated.
+      # Assigning a new value to a readonly attribute raises an error by default.
+      # By setting `config.active_record.raise_on_assign_to_attr_readonly` to `false` it will not
+      # raise and ignore the attribute instead.
```

Comment:
Thanks for following up on this!

I think for both this paragraph and the one before it may be worth clarifying that "ignore these fields" and "ignore the attribute" actually means that the value will update in memory but not persist to the database.

Reply:
Good call. I reworded to not use the word "ignore".